firmware/CMakeLists.txt

```diff
@@ -71,14 +71,45 @@ PRIVATE
 	${CMAKE_CURRENT_SOURCE_DIR}  # provides app.h
 )
 
+target_include_directories(os
+PRIVATE
+	${CMAKE_CURRENT_SOURCE_DIR}  # provides app.h
+)
+
 target_link_libraries(main
 PRIVATE
 	bindings
 )
 
+target_link_libraries(main
+PRIVATE
+	os
+)
+
 set(POSTBUILD_FILE ${DIR_MCAL_PLATFORM}/PostBuild.cmake)
 if(EXISTS "${POSTBUILD_FILE}" AND NOT IS_DIRECTORY "${POSTBUILD_FILE}")
 	include("${POSTBUILD_FILE}")
 else()
 	message(FATAL_ERROR "${POSTBUILD_FILE} does not exist. If no postbuild actions are required, create the file and leave it empty.")
-endif()
\ No newline at end of file
+endif()
+
+# FreeRTOS code - hack for now
+set(FREERTOS_INCLUDE_DIRECTORIES
```

Comment:
Should be using target_link_directories, not variables. Please fix this "hack"

Reply:
will remove this

Reply:
Removed this legacy code, it was not used anywhere.